kubernetes/kustomize/overlays/production/tls-secret.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Can you rearrange entries? Like 
apiVersion
kind
metadata
type
data

I think it's more readable if it's unified with the rest of the files